DZSwiftUI/DZSwiftUI/VPNView.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  VPNView.swift
+//  DZSwiftUI
+//
+//  Created by Maxim on 21.01.2023.
+//
+
+import SwiftUI
+
+/// Экран запуска VPN
+struct VPNView: View {
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let spacingValue = 20.0
+        static let countryImageName = "singapur"
+        static let vpnToggleOffText = "OFF"
+        static let vpnToggleOffImageName = "poweroff"
+        static let vpnToggleOnText = "ON"
+        static let vpnToggleOnImageName = "poweron"
+        static let vpnToggleCapsuleFrameWidth = 80.0
+        static let vpnToggleCapsuleFrameHeight = 44.0
+        static let vpnToggleCircleFrameWidth = 40.0
+        static let vpnToggleOffsetX = 18.0
+        static let vpnDefaultTextFrameWidth = 120.0
+        static let vpnIPAddressText = "IP Address"
+        static let vpnIPAddressValue = "127.0.0.1"
+        static let vpnReceivedText = "Received"
+        static let vpnReceivedValue = "0 MB"
+        static let vpnSentText = "Sent"
+        static let vpnSentValue = "0 MB"
+        static let vpnInfoText = "Service will expire after 15 days."
+        static let pickerValueStart = 0
+        static let vpnSelectCountryPickerFrameHeight = 100.0
+    }
+    
+    // MARK: - Body
+    
+    var body: some View {
+        VStack(spacing: Constants.spacingValue) {
+            vpnCountryTextView
+            countryImageView
+            vpnSelectCountryPickerView
+            vpnToggleView
+            vpnIPTextView
+            vpnReceivedTextView
+            vpnSentTextView
+            Spacer()
+            vpnInfoTextView
+        }
+    }
+    
+    // MARK: - Private Properties
+    
+    @EnvironmentObject private var vpnViewModel: VPNViewModel
+    @State private var isOnVPNToggleShown = false
```

Comment:
мне все-таки кажется, что он всегда shown и это слово надо убрать
```suggestion
    @State private var isOnVPNToggle = false
```